### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Adding Code of Conduct to project.

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (emilyrohrbough) [inline CODE_OF_CONDUCT.md]: pledge to make*

Comment (bjankord) [inline CODE_OF_CONDUCT.md]: Updated here: https://github.com/cerner/terra-core/pull/503/commits/976a54fc48e4b2d0308bea1eadacfc264ea79221

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
```
Comment (emilyrohrbough) [inline CODE_OF_CONDUCT.md]: insulting or derogatory

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
```
Comment (emilyrohrbough) [inline CODE_OF_CONDUCT.md]: Code of Conduct. We also have the right to temporarily or permanently ban

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at DL_TERRA_CORE_TEAM@cerner.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
```
Comment (emilyrohrbough) [inline CODE_OF_CONDUCT.md]: needs a . at the end

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Matt-Butler) [inline CODE_OF_CONDUCT.md]: This COC is a standard template that comes from here: https://github.com/cerner/terra-core/community/code-of-conduct/new?template=contributor-covenant

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (bjankord) [inline CODE_OF_CONDUCT.md]: Updated here: https://github.com/cerner/terra-core/pull/503/commits/e3bcd268efcc0b3577677e300a75c337279702ae

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (bjankord) [inline CODE_OF_CONDUCT.md]: Updated here: https://github.com/cerner/terra-core/pull/503/commits/e3bcd268efcc0b3577677e300a75c337279702ae

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (bjankord) [inline CODE_OF_CONDUCT.md]: Updated here: https://github.com/cerner/terra-core/pull/503/commits/e3bcd268efcc0b3577677e300a75c337279702ae

**Pull request merged**